server/render/index.js

```diff
@@ -68,12 +67,18 @@ export function render( element, key = JSON.stringify( element ) ) {
 	//todo: render an error?
 }
 
-export function serverRender( context ) {
-	if ( config.isEnabled( 'server-side-rendering' ) ) {
+export function serverRender( req, res ) {
```

Comment:
I was really talking about the name of the function. `serverRender` sounds like it has something to do with SSR, which this function doesn't necessarily.